compiler/src/main/java/com/squareup/anvil/compiler/codegen/BindingModuleGenerator.kt

```diff
@@ -73,7 +71,7 @@ internal class BindingModuleGenerator(
 
   private val contributedBindingClasses = mutableListOf<FqName>()
   private val contributedMultibindingClasses = mutableListOf<FqName>()
```

Comment:
Could these be converted to `ClassReference` too? Now that we have `ClassReference` working with `FqName` is bad practice, it's a very weak type.